Readme.md

```diff
@@ -1,6 +1,11 @@
 # A writer for object file ar archives
 
-This is based on commit [8ef3e895a](https://github.com/llvm/llvm-project/tree/3d3ef9d073e1e27ea57480b371b7f5a9f5642ed2) (15.0.0-rc3) of LLVM's archive writer.
+This is a Rust port of LLVM's archive writer (`ArchiveWriter.cpp`):
+* Based on commit [8ef3e895a](https://github.com/llvm/llvm-project/tree/3d3ef9d073e1e27ea57480b371b7f5a9f5642ed2) (15.0.0-rc3).
+* With the following options removed:
+  * Deterministic: always enabled.
+  * Thin: always disabled.
+  * Symbol tables: always enabled.
```

Comment:
Always enabling deterministic mode makes sense to me. Symbol tables can still be disabled by passing in a `get_symbols` function which returns no symbols at all, so removing that option is fine too. As for thin archives however, I can imagine that rustc may add support for writing them in the future. As such I prefer if support for thin archives is kept.

Reply:
Done